deploy-board/deploy_board/templates/clusters/clusters.html

```diff
@@ -47,6 +65,13 @@ <h4 class="panel-title pull-left">Cluster Manager</h4>
 {% endblock %}
 
 {% block main %}
+{% if not basic_cluster_info|canReplaceCluster %}
+    <div class="alert alert-warning" role="alert">
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <strong>Warning!</strong> This environment stage has scheduled cluster rolling upgrade.
```

Comment:
Since we think the cluster replace should be be interfering with regular deploy - I would suggest change this to info ( light blue)